src/services/Content/ContentQueries.ts

```diff
@@ -35,14 +35,19 @@ export const useAddContentQuery = (queryClient: QueryClient, request: CreateCont
   const { notify } = useNotification();
   return useMutation(() => AddContentListItems(request), {
     onSuccess: () => {
+      const title =
+        request?.length > 1
+          ? `${request?.length} custom repositories added`
+          : `Custom repository "${request?.[0]?.name}" added`;
       notify({
         variant: AlertVariant.success,
-        title: `Successfully added ${request.length} ${request.length > 1 ? 'items' : 'item'}.`,
+        title,
+        description: 'The validation and package introspection process may take some time',
```

Comment:
Thinking about this message, I'm not sure which validation process would take some time? I know the mock says this, I'm just wondering if that messaging is still accurate. Also, I'm not sure if you'd say "package introspection" instead of "repository introspection". Maybe @jlsherrill can chime in here.

Reply:
yes, i'd say "repository introspection" and i also would not mention validation taking some time.

Reply:
Roger